external_dns/assets/service_checks.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
```suggestion
[
    {
        "agent_version": "6.15.0",
        "integration": "External DNS",
        "check": "external_dns.prometheus.health",
        "statuses": ["ok", "critical"],
        "groups": ["host", "endpoint"],
        "name": "External DNS prometheus health",
        "description": "Returns `CRITICAL` if the check cannot access the metrics endpoint, or `OK` otherwise."
    }
]
```